### PR TITLE
Bug 1132929 - [Accessibility] fixing visibility issues around Video app.

### PR DESCRIPTION
**Pull request by yzen**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=568ab3e80839)

File: apps/video/index.html [hunk 1]
```diff
@@ -51,68 +51,70 @@
     <script defer src="js/video.js"></script>
   </head>
   <body role="application" class="skin-dark theme-media">
-    <div id="two-column-spearator"></div>
-    <section role="region" id="thumbnail-list-view">
-      <gaia-header id="picker-header" action="back">
-        <h1 id="picker-title" data-l10n-id="pick-title"></h1>
-        <h1 id="thumbnail-list-title">Video</h1>
-      </gaia-header>
+    <div id="container">
```
Comment (russnicoletti) [inline apps/video/index.html]: Nit: The readability of the html would be improved by using a more descriptive name for "container". Some possibilities: "thumbnail-player-container", "non-overlay-container", "app-container".

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f3825ec9efc7)

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ba8e295e4004)

**Pull request merged**